composer.json

```diff
@@ -36,13 +42,14 @@
 		"microsoft/azure-storage-blob": "^1.5",
 		"psr/log": "^1.1|^2.0|^3.0",
 		"symfony/filesystem": "^6.0||^5.0||^4.0",
-		"symfony/process": "^6.0||^5.0||^4.0"
+		"symfony/process": "^6.0||^5.0||^4.0",
+		"tomasfejfar/phpstan-phpunit": "dev-first"
```

Comment:
haha, jsem si nejdirv myslel, ze to je nejakej zase composer specialni pripad, ale ona se jen ta branch jmenuje `first` :D